consensus/wal.go

```diff
@@ -106,8 +106,8 @@ func (wal *baseWAL) OnStart() error {
 }
 
 func (wal *baseWAL) OnStop() {
-	wal.BaseService.OnStop()
 	wal.group.Stop()
+	wal.group.Close()
```

Comment:
why not make this part of Stop ?

Reply:
To keep the parity: OpenGroup/Close, Start/Stop